common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +267,63 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    const { to, value } = this.state;
+    return (
+      isValidETHAddress(to) &&
+      value &&
+      Number(value) > 0 &&
+      !isNaN(Number(value)) &&
+      isFinite(Number(value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState() {
+    // FIXME add gas price
+    if (this.state.unit === 'ether') {
+      return {
+        to: this.state.to,
+        from: this.props.wallet.getAddress(),
+        // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
+        value: '0x' + toHex(toWei(Big(this.state.value), 'ether'))
```

Comment:
woujld prefer to see this constructed step by step outside of the component, perhaps in a help utility, with a comment on each line describing why we do each step

e.g. 

// Convert to Big because of the size of the value.
```
valueToHex = function(value, kind) {
    // To Big because of the size of the value
    const bigValue = Big(value)
    // To Wei because 
    ... 

}
```
etc.